node/src/node_configurator/configurator.rs

```diff
@@ -2620,6 +2622,119 @@ mod tests {
         assert_eq!(*past_neighbors_params, vec!["password".to_string()])
     }
 
+    #[test]
+    fn configuration_handles_retrieving_max_block_count_none() {
```

Comment:
Hmm. Matter-of-factly, if you look at the trails in the method, there is always just one, no matter whether the value for the max_block_count evaluates to None or Some(). That said, you have already tested this new piece of code by the two preexisting tests where you've added the parameter in. I consider this test groundless.

You either should delete it or if you do want to keep it, I would recommend to rewrite it into a more general variation of itself where you'd be claiming on testing all optional parameters when they return None. I'd probably understand that but otherwise this test shouldn't stay. (It's also noticeably inconsistent with the other picks of tests where you could have the same kind of argument, which you might have had in your mind, for every other optional value in there but yet such tests don't exist.)

Reply:
ok, deleted